src/index.js

```diff
@@ -92,13 +92,16 @@ class CID {
     return this.buffer
   }
 
+  toV0String () {
+    return mh.toB58String(this.multihash)
```

Comment:
this should error if it's a V1 cid. we dont want people pointing to git objects with just a multihash.